packages/api-server/src/createServer.ts

```diff
@@ -302,32 +304,3 @@ export const DEFAULT_CREATE_SERVER_OPTIONS: DefaultCreateServerOptions = {
     requestTimeout: 15_000,
   },
 }
-
-export interface RedwoodFastifyAPIOptions {
-  redwood: {
-    apiRootPath: string
-  }
-}
-
-export async function redwoodFastifyFunctions(
-  fastify: FastifyInstance,
-  opts: RedwoodFastifyAPIOptions
-) {
-  fastify.register(fastifyUrlData)
-  await fastify.register(fastifyRawBody)
-
-  fastify.addContentTypeParser(
-    ['application/x-www-form-urlencoded', 'multipart/form-data'],
-    { parseAs: 'string' },
-    fastify.defaultTextParser
-  )
-
-  fastify.all(`${opts.redwood.apiRootPath}:routeName`, lambdaRequestHandler)
-  fastify.all(`${opts.redwood.apiRootPath}:routeName/*`, lambdaRequestHandler)
-
-  await loadFunctionsFromDist({
-    fastGlobOptions: {
-      ignore: ['**/dist/functions/graphql.js'],
-    },
-  })
-}
```

Comment:
No more duplication internally anymore either; createServer uses the same plugin apiServerHandler does, just with some fast glob options to ignore the graphql function so it can register it itself